File: 0027-remove-element/0027-remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        v = [num for num in nums if num != val]
        for i in range(len(v)):
            nums[i] = v[i]
        return len(v)
        

File: 0027-remove-element/0027-remove-element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int n = nums.size();
        vector<int>v;
        for(int i=0; i<n;i++){
            if(nums[i] != val) v.push_back(nums[i]);
        }

        int m = v.size();
        for(int i=0;i<m;i++){
            nums[i] = v[i];
        }
        return m;
    }
};